docs/production-deployment/self-hosted-guide/temporal-nexus.mdx

```diff
@@ -11,6 +11,27 @@ tags:
   - enable-nexus
 ---
 
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+Learn why you should use Nexus in the [evaluation guide](/evaluate/nexus).
+
+:::
+
+[Temporal Nexus](/nexus) allows you to reliably connect Temporal Applications.
+It was designed with Durable Execution in mind and enables each team to have their own Namespace for improved modularity, security, debugging, and fault isolation.
+
+![Nexus Overview](/img/nexus-overview-short.png)
+
+:::tip RELATED
+
+- [Evaluate](/evaluate/nexus) why you should use Nexus and learn more about [Nexus use cases](/evaluate/nexus#use-cases).
+- [Learn Nexus concepts](/nexus) in the Encyclopedia.
```

Comment:
I see this tip appears at the bottom of this page again in a slightly different form. I would fix this.